ajax/kanban.php

```diff
@@ -118,6 +118,19 @@
    $inputs = [];
    parse_str($_REQUEST['inputs'], $inputs);
    $item->add($inputs);
+} else if ($_REQUEST['action'] == 'bulk_add_item') {
+   $checkParams(['inputs']);
+   $item = new $itemtype();
+   $inputs = [];
+   parse_str($_REQUEST['inputs'], $inputs);
+
+   $bulk_item_list = preg_split('/\r\n|[\r\n]/', $inputs['bulk_item_list']);
+   if (!empty($bulk_item_list)) {
+      unset($inputs['bulk_item_list']);
+      foreach ($bulk_item_list as $item_entry) {
+         $item->add($inputs + ['name' => $item_entry]);
```

Comment:
You should add a test to prtevent created items from empty lines
```suggestion
         $item_entry = trim($item_entry);
         if (!empty($item_entry)) {
            $item->add($inputs + ['name' => $item_entry]);
         }
```